### ISSUE TITLE
Pretrain Hubert base second iteration

### ISSUE THREAD

**Issue by renadnasser1** [facebookresearch/fairseq#5097]
I'm training a Hubert model from scratch on 8k Hz audio speech data same as described on the paper, first iteration succeeded.  I've started the second iteration where first iteration features were used to learns kmeans clusters.
why the follow warning printed for all the training data. should I be concerned ?

`[2023-05-02 16:53:22,434][fairseq.data.audio.hubert_dataset][WARNING] - audio and label duration differ too much 
`

**Comment by MorenoLaQuatra** [comment]
I don't know if it is relevant anymore, anyway, I think you need to set `model.label_rate=50` for the second iteration.

**Comment by GUOhm230** [comment]
How do you set up a single machine with multiple GPU?

**Comment by GUOhm230** [comment]
> tarted the second iteration where first iteration features were used to learns kmeans clusters.
> why the follow warning printed for all the training data. should I be concerned ?

Do you need to use the model parameters of the first training for the second iterative training?
